src/main/java/seedu/ui/Help.java

```diff
@@ -4,15 +4,17 @@
 
     public void printGetHelp() {
         StringBuilder output = new StringBuilder(
-            "Hello, please select 1-4 for its command format.\n"
+            "Hello, please select 1-5 for its command format.\n"
             + "1. Student List\n"
             + "2. Event-related\n"
             + "3. Calendar\n"
-            + "4. Attendance\n" + "5. Peformance\n"
+            + "4. Attendance\n"
+            + "5. Performance\n"
         );
-        output.append("To track any list, input: <type_of_list> list\n");
-        output.append("Note: All command are NOT case sensitive.\n");
-
+        output.append("To view event list, input: event list\n");
```

Comment:
For the output.append, why are we appending multiple lines instead of 1 long string with "\n"?

Reply:
im following jia yee's way, might need to ask him for this